phpBB/styles/prosilver/template/posting_buttons.html

```diff
@@ -105,6 +105,16 @@
 	<button type="button" class="button button-icon-only bbcode-color" name="bbpalette" id="bbpalette" value="{L_FONT_COLOR}" onclick="change_palette();" title="{L_BBCODE_S_HELP}">
 		<i class="icon fa-tint fa-fw" aria-hidden="true"></i>
 	</button>
+	<!-- IF S_SMILIES_ALLOWED -->
+	<button type="button" class="button button-icon-only smiley_button" accesskey="y" name="addsmiley" title="{L_SMILEYS}">
```

Comment:
button shouldnt be inside a  div block. Because that button also should be in the same row of other contents. So I used this.
`<!-- IF S_SMILIES_ALLOWED -->
	<button type="button" class="button button-icon-only smiley_button" accesskey="y" name="addsmiley" title="{L_SMILEYS}">
		<div class="dropdown-container smiley-container" data-skip-responsive="true">
			<a class="dropdown-trigger" title="{L_SMILEYS}">
				<i class="icon fa-smile-o fa-fw"></i>
			</a>
			<!-- INCLUDE smiley_dropdown.html -->
		</div>
	</button>`
 
I have checked the codes of both quick link dropdown and notification dropdown. 

`<div id="smiley-dropdown" class="smiley-dropdown dropdown" >
        <div class="pointer"><div class="pointer-inner"></div></div>
        <div class="dropdown-contents smiley-content">
                       // smiley box content.
        </div>
</div>`

In the notification dropdown code was as
`<div id="notification_list" class="dropdown dropdown-extended notification_list">
	<div class="pointer"><div class="pointer-inner"></div></div>
	<div class="dropdown-contents">
                  //notification-content.
    </div>
</div>`

In here for the notification dropdown "dropdown-extended" class is used which is an extension of dropdown class.
So for the smiley dropdown I have used "smiley-content" class which is also an extension of dropdown class.